packages/react/src/components/DateTimePicker/DateTimePickerV2.jsx

```diff
@@ -0,0 +1,988 @@
+  light: PropTypes.bool,
+  /** The language locale used to format the days of the week, months, and numbers. */
+  locale: PropTypes.string,
+  /** Unique id of the component */
+  id: PropTypes.string,
+  /** Optionally renders only an icon rather than displaying the current selected time */
+  hasIconOnly: PropTypes.bool,
+};
+
+const defaultProps = {
+  testId: 'date-time-picker',
+  defaultValue: null,
+  dateTimeMask: 'YYYY-MM-DD HH:mm',
+  presets: PRESET_VALUES,
+  intervals: [
+    {
+      label: 'minutes',
+      value: INTERVAL_VALUES.MINUTES,
+    },
+    {
+      label: 'hours',
+      value: INTERVAL_VALUES.HOURS,
+    },
+    {
+      label: 'days',
+      value: INTERVAL_VALUES.DAYS,
+    },
+    {
+      label: 'weeks',
+      value: INTERVAL_VALUES.WEEKS,
+    },
+    {
+      label: 'months',
+      value: INTERVAL_VALUES.MONTHS,
+    },
+    {
+      label: 'years',
+      value: INTERVAL_VALUES.YEARS,
+    },
+  ],
+  relatives: [
+    {
+      label: 'Today',
+      value: RELATIVE_VALUES.TODAY,
+    },
+    {
+      label: 'Yesterday',
+      value: RELATIVE_VALUES.YESTERDAY,
+    },
+  ],
+  expanded: false,
+  disabled: false,
+  showRelativeOption: true,
+  showCustomRangeLink: true,
+  hasTimeInput: true,
+  renderPresetTooltipText: null,
+  onCancel: null,
+  onApply: null,
+  i18n: {
+    toLabel: 'to',
+    toNowLabel: 'to Now',
+    calendarLabel: 'Calendar',
+    presetLabels: [
+      'Last 30 minutes',
+      'Last 1 hour',
+      'Last 6 hours',
+      'Last 12 hours',
+      'Last 24 hours',
+    ],
+    intervalLabels: ['minutes', 'hours', 'days', 'weeks', 'months', 'years'],
+    relativeLabels: ['Today', 'Yesterday'],
+    customRangeLinkLabel: 'Custom Range',
+    customRangeLabel: 'Custom range',
+    relativeLabel: 'Relative',
+    lastLabel: 'Last',
+    invalidNumberLabel: 'Number is not valid',
+    relativeToLabel: 'Relative to',
+    absoluteLabel: 'Absolute',
+    startTimeLabel: 'Start time',
+    endTimeLabel: 'End time',
+    applyBtnLabel: 'Apply',
+    cancelBtnLabel: 'Cancel',
+    backBtnLabel: 'Back',
+    increment: 'Increment',
+    decrement: 'Decrement',
+    hours: 'hours',
+    minutes: 'minutes',
+    number: 'number',
+  },
+  light: false,
+  locale: 'en',
+  id: undefined,
+  hasIconOnly: false,
+};
+
+const DateTimePicker = ({
+  testId,
+  defaultValue,
+  dateTimeMask,
+  presets,
+  intervals,
+  relatives,
+  expanded,
+  disabled,
+  showRelativeOption,
+  showCustomRangeLink,
+  hasTimeInput,
+  renderPresetTooltipText,
+  onCancel,
+  onApply,
+  i18n,
+  light,
+  locale,
+  id = uuid.v4(),
+  hasIconOnly,
+  ...others
+}) => {
+  React.useEffect(() => {
+    if (__DEV__) {
+      warning(
+        false,
+        'The `ColorDropdown` is an experimental component and could be lacking unit test and documentation. Be aware that minor version bumps could introduce breaking changes. For the reasons listed above use of this component in production is highly discouraged'
```

Comment:
Did we mean to leave this text in the warning here?

Reply:
I think this should be DateTimePickerV2 instead of ColorDropdown. Does the DateTimePickerV2 need to be experimental?

Reply:
fixed